src/util.h

```diff
@@ -32,6 +32,7 @@
 
 #include <stdint.h>
 #include "sds.h"
+#include "monotonic.h"
```

Comment:
`monotonic.h` doesn't need to be included here.  We should limit includes to just what is needed for the `.h` file to compile.

Reply:
@xbasel can we remove this?
```suggestion
```